src/lib/tool_providers.ml

```diff
@@ -494,7 +506,7 @@ let default_toolkit
     star_tool ~host ~meta_playground;
     stringtie_tool ~host ~meta_playground;
     cufflinks_tools ~host ~meta_playground;
-    hisat_tool ~host ~meta_playground;
+    hisat_tool ~host ~meta_playground ~version:`V_0_1_6_beta;
```

Comment:
I guess the question is should there be a 1-1 mapping between the default tools and the default toolkit - if so, how is it enforced